src/lean_spec/subspecs/containers/signature.py

```diff
@@ -0,0 +1,62 @@
+"""Signature container."""
+
+from __future__ import annotations
+
+from lean_spec.types.collections import SSZList, SSZVector
+from lean_spec.types.container import Container
+from lean_spec.types.uint import Uint8
+
+MAX_TREE_DEPTH = 32
+
+
+class HashDigest(SSZVector):
+    """A 32-byte hash digest."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
+
+
+class Parameter(SSZVector):
+    """A 32-byte parameter."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
+
+
+class Randomness(SSZVector):
+    """A 32-byte randomness seed."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
```

Comment:
This one would be

```suggestion
    LENGTH = XmssConfig.RAND_LEN_FE * BYTES_PER_FE
```

ref: https://github.com/leanEthereum/leanSpec/blob/main/src/lean_spec/subspecs/xmss/constants.py#L72

Reply:
@varun-doshi Sorry I know we chatted about these fixed values earlier, but this idea of referring to xmss subspecs instead came after I read the PR again